### ISSUE TITLE
version `GLIBC_2.28' not found

### ISSUE THREAD

**Issue by WangXiangUSTC** [chaos-mesh/chaosd#61]
```bash
./chaosd --help
./chaosd: /lib64/libc.so.6: version `GLIBC_2.28' not found (required by ./chaosd)
```

**Comment by WangXiangUSTC** [comment]
it is because that github.com/chaos-mesh/chaos-mesh/pkg/ptrace need to use cgo

**Comment by WangXiangUSTC** [comment]
the sqlite also require cgo

**Comment by WangXiangUSTC** [comment]
since pr #74 merged, chaosd require at least v2.17, and it is suitable for most Linux versions.
